_episodes/01-introduction.md

```diff
@@ -136,7 +120,6 @@ Situated learning thus describes why we teach,
 and recognizes that teaching and learning is necessarily rooted in a social context.
 We then depend on the cognitivist perspective
 to drive *how* we teach the specific content associated with the community of practice.
-Many of our teaching techniques and lesson development strategies incorporate ideas from this perspective.
```

Comment:
Why remove? Live coding and the use of GitHub for building workshop websites are both directly motivated by the notion of welcoming someone into a community of practice.  (Or should the sentence be relocated because it's ambiguous where it is?)